easybuild/easyconfigs/m/MATLAB-Engine/MATLAB-Engine-2021a-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'PythonPackage'
+
+name = 'MATLAB-Engine'
+version = '2021a'
```

Comment:
i actually messed up here, pretty sure i meant for this to be 2021b, which should be compatible with python 3.9 out of the box if i recall correctly.

Reply:
Actually I'm interested in making this work with our Python in 2021a and I think I got a workable fix. However, I'm hitting another error in both Python 3.9 and 3.8 where the engine libs are not found. Not sure what is happening yet

Reply:
Is there any reason to think that would work?

Also, i think it would be more appropriate for the for this to just be the 2021b version of matlab as the default, since 2021b version of matlab happens to match the GCCcore 11.2.0 toolchain release.

and it this needs to be backported to an older matlab for a newer toolchain then add a suffix for that special case

Reply:
yes, having version 2021b in GCCcore/11.2.0 makes most sense.

I'm dealing with this issue now:
https://www.mathworks.com/matlabcentral/answers/597865-matlab-linking-error-centos-7-r2020b#answer_1007455

Apparently in Centos7 the engines cannot be loaded out-of-box